### ISSUE TITLE
Robustness plot differs from t-test to summary t-test

### ISSUE THREAD

**Issue by lindeloev** [jasp-stats/jasp-desktop#1810]
## Environment (for bugs)

* JASP version: 0.8.1.1
* OS name and version: Ubuntu 16.04

The robustness figure for summary t-test does not include points marking "wide" and "ultrawide" priors (see figure below). The classical t-test robustness figure does. Not a big issue, I just noticed the inconsistency. Worse, it implies duplicated underlying code?
![image](https://user-images.githubusercontent.com/1439417/27035680-7ec8b462-4f82-11e7-922b-57141bd3ac3b.png)

Another small idea: perhaps add a point for the "normal" or "default" prior if the user specifies something else.

**Comment by EJWagenmakers** [comment]
@akashrajkn @quentingronau @AlexanderLyNL 
Did you tick "additional info"? I agree that JASP Common and JASP SumStats are inconsistent wrt the robustness check. I actually prefer the SumStats version, with the additional information available upon demand. I hope we can upgrade the robustness checks in JASP Common in one of our upcoming new versions.
E.J.

**Comment by akashrajkn** [comment]
@EJWagenmakers I'll work on having the additional information available for all robustness plots.

**Comment by lindeloev** [comment]
@EJWagenmakers Yes, you are right that the extra info appears upon ticking "additional info", my bad. So yes, it's the presence of the "additional info" box and it's behavior which differs between summary- and data-t-test. 

There's also a "max BF" in the summary, not present which in the t-test. I find that it almost encourages defining priors post-hoc. Maybe it's a honey trap? ;-)

**Comment by EJWagenmakers** [comment]
@lindeloev The Max BF is there to show that the evidence doesn't get any better, even if you cherry-pick the prior. Maybe another example how any statistical method can be abused :-)
